policy/predicate/file.go

```diff
@@ -184,16 +241,50 @@ func (pred *ModifiedLines) Evaluate(ctx context.Context, prctx pull.Context) (bo
 		deletions += int64(f.Deletions)
 	}
 
-	for expr, v := range map[ComparisonExpr]int64{
-		pred.Additions: additions,
-		pred.Deletions: deletions,
-		pred.Total:     additions + deletions,
-	} {
-		if !expr.IsEmpty() && expr.Evaluate(v) {
-			return true, "", nil
+	if !pred.Additions.IsEmpty() && pred.Additions.Evaluate(additions) {
+		res, err := pred.Additions.MarshalText()
```

Comment:
It's unfortunate that you need to check the error here, especially since it should always be `nil` in practice. I think it would make sense to:

1. Add a `String` method to `ComparisonExpr`. This will basically be the same implementation as `MarshalText`, but it can use `?` or `_` or `<unknown>` if the operator is not supported instead of returning an error.
2. Modify `MarshalText` to call `String` (or possibly remove it completely)
3. Use `String` in this function to remove all the error handling

Reply:
Fixed this by creating a function called `MarshalTextToString` that runs the steps you mentioned above.